Merge/mainnet-readiness.md

```diff
@@ -127,6 +134,9 @@ Most research has been completed. Only listing things still left to dig deeper i
     * [ ] Sync during transition period
 * [ ] Discovery [is there actually anything to do here?]
 * [ ] Execution-layer proof of custody
+* [ ] Disaster recovery if invalid chain finalized 
+  * [x] [WIP] Client multiplexers ([link](https://github.com/karalabe/minority))
 * [ ] Further threat analysis
     * [ ] Miner attacks
     * [ ] Resource exhaustion post-merge
+* [ ] Fee Market behavior changes (missed slots impact)
```

Comment:
I'm not sure this is relevant for the merge itself,.

Reply:
I'd like to see an EIP specifying this soon, knowing that it likely won't go into the merge

Reply:
cc: @adietrichs :-)